lib/Dialects/LinalgTransform/IR/LinalgTransformOps.cpp

```diff
@@ -7,17 +7,51 @@
 //===----------------------------------------------------------------------===//
 
 #include "Dialects/LinalgTransform/LinalgTransformOps.h"
+#include "Dialects/LinalgTransform/ScopedTransform.h"
```

Comment:
Note: I haven't reviewed this file yet as changes seem to be mostly moves.
Since github does not have the nice phab-like "code was moved" highlighting, I am skipping for now.

Please lmk if this is fundamentally more than a move and I'll look deeper.